lib/Fhp/Dialog/Dialog.php

```diff
@@ -419,7 +417,7 @@ public function initDialog()
 	 * @throws FailedRequestException
 	 * @throws \Exception
 	 */
-	public function syncDialog($sendHKTan = true, $tanMechanism = null)
+	public function syncDialog($tanMechanism = null, $tanMediaName = null)
```

Comment:
hktan is only sent if a `$tanMechanism` is provided

Reply:
:heavy_check_mark: